src/dependency.rs

```diff
@@ -18,6 +18,12 @@ pub struct Dependency {
     pub kind: Kind,
 }
 
+pub struct ReverseDependency {
```

Comment:
Mostly just curious, what was it about clippy that made this new `ReverseDependency` struct needed?